src/elements/content-sidebar/versions/VersionsSidebar.js

```diff
@@ -21,19 +22,31 @@ const MAX_VERSIONS = DEFAULT_FETCH_END;
 
 type Props = {
     error?: MessageDescriptor,
+    errorTitle?: MessageDescriptor,
     fileId: string,
     isLoading: boolean,
+    onUpgradeCTAClick: () => void,
     parentName: string,
+    showVersionHistoryErrorUpsell: boolean,
```

Comment:
This is an optional prop, right?

```suggestion
    showVersionHistoryErrorUpsell?: boolean,
```

Reply:
Looking back through, I guess this probably isn't. Maybe at the container layer, but not here.

Reply:
yup, it is optional at the container, but always sets it here. Is that ok? Or should I make it optional everywhere?

Reply:
Actually I removed this here based on @jstoffan suggestion below.